content/doc/book/updating/linux.adoc

```diff
@@ -0,0 +1,108 @@
+---
+layout: section
+title: Linux
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc: left
+:toclevels: 4
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+Linux installations should be upgraded using the same method as was used for the original installation
+This section will guide you through updating your Jenkins instance to the most recent version.
+
+include::doc/book/updating/_pre-upgrade-wizard.adoc[]
+
+== Upgrade steps on Linux
+
+=== 1.Stop your Jenkins Instance
+
+[source,bash]
+```
+#Stop running Jenkins service using the following command:
+
+sudo su -
+service jenkins stop
+```
+
+=== 2.  Take Jenkins instance Backup as a recovery option
+[source,bash]
+```
+#Take backup of Jenkins home directory. In my case my Jenkins home directory is /var/lib/jenkins.
+
+cd /var/lib
+tar -cvzf jenkins_date.tar.gz jenkins/
+mv jenkins_date.tar.gz $HOME
+```
+
+=== 3. Take The Backup Of the Current Jenkins Version Binary
+[source,bash]
+```
+#Take the backup of Jenkins current version binary using following commands:
+
+cd /usr/share/jenkins
+mv jenkins.war jenkins.war.old
+```
+
+=== 4. Download The New LTS Jenkins Version
+[source,bash]
+```
+#Download the new version of Jenkins WAR file using the following commands:
+
+wget https://updates.jenkins-ci.org/latest/jenkins.war
+```
+
+=== 5. Start The Jenkins Service
+[source,bash]
+```
+#Start the Jenkins service using the following command:
+
+service jenkins start
+```
+
+# Based on Operating Systems
```

Comment:
```suggestion
#Based on Operating Systems
```

Reply:
This is a heading 
that's why we can't remove the space

Reply:
I am not sure, but I don't think you are supposed to have any 0 level sections here

Reply:
@Vandit1604 Are you sure you are supposed to use a # for the heading, I see you using the syntax `#heading` in a few instances in the same file above.